code/modules/language/swarmer.dm

```diff
@@ -0,0 +1,26 @@
+/datum/language/swarmer
+	name = "Swarmer"
+	desc = "A language only consisting of musical notes."
+	speech_verb = "tones"
+	ask_verb = "tones inquisitively"
+	exclaim_verb = "tones loudly"
+	colour = "changeling"
+	key = "s"
+	flags = NO_STUTTER
+	space_chance = 100
+	sentence_chance = 0
+	// since various flats and sharps are the same,
+	// all non-accidental notes are doubled in the list
+	syllables = list(
```

Comment:
Sadly, I'm not sure if Byond can handle the unicode characters, so I might have to resort to `#` and `b`, which depending on font, doesn't look as good.